packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/modal/ReactModalHostView.java

```diff
@@ -331,9 +334,34 @@ public boolean onKey(DialogInterface dialog, int keyCode, KeyEvent event) {
       mDialog.show();
       if (context instanceof Activity) {
         if (Build.VERSION.SDK_INT > Build.VERSION_CODES.R) {
-          int appearance =
-              ((Activity) context).getWindow().getInsetsController().getSystemBarsAppearance();
-          mDialog.getWindow().getInsetsController().setSystemBarsAppearance(appearance, appearance);
+          Window window = ((Activity) context).getWindow();
```

Comment:
Would a window's properties of showing status bar, etc. ever change? I guess I'm wondering, does it make sense to put this logic under `updateProperties`?